crates/rolldown/src/ast_scanner/mod.rs

```diff
@@ -590,11 +598,8 @@ impl<'me> AstScanner<'me> {
         }
       }
       None => {
-        if ident.name == "module" {
-          self.cjs_module_ident.get_or_insert(Span::new(ident.span.start, ident.span.start + 6));
-        }
-        if ident.name == "exports" {
-          self.cjs_exports_ident.get_or_insert(Span::new(ident.span.start, ident.span.start + 7));
+        if !self.is_cjs_module {
+          self.is_cjs_module = ident.name == "module" || ident.name == "exports";
```

Comment:
global scoped `module` and `exports` appear in anywhere is considered as a commonjs module.